src/azure-cli/azure/cli/command_modules/sql/_params.py

```diff
@@ -1483,22 +1483,124 @@ def _configure_security_policy_storage_params(arg_ctx):
+    with self.argument_context('sql midb short-term-retention-policy show') as c:
+        c.argument(
+            'deleted_time',
+            options_list=['--deleted-time'],
+            help='If specified, shows retention days for a deleted database, instead of an existing database.'
+            'Must match the deleted time of a deleted database on the source Managed Instance.')
+
+    with self.argument_context('sql midb ltr-policy set') as c:
+        create_args_for_complex_type(
+            c, 'parameters', ManagedDatabase, [
+                'weekly_retention',
+                'monthly_retention',
+                'yearly_retention',
+                'week_of_year'
+            ])
+
+        c.argument('weekly_retention',
+                   help='Retention for the weekly backup. '
+                   'If just a number is passed instead of an ISO 8601 string, days will be assumed as the units.'
+                   'There is a minimum of 7 days and a maximum of 10 years.')
+
+        c.argument('monthly_retention',
+                   help='Retention for the monthly backup. '
+                   'If just a number is passed instead of an ISO 8601 string, days will be assumed as the units.'
+                   'There is a minimum of 7 days and a maximum of 10 years.')
+
+        c.argument('yearly_retention',
+                   help='Retention for the yearly backup. '
+                   'If just a number is passed instead of an ISO 8601 string, days will be assumed as the units.'
+                   'There is a minimum of 7 days and a maximum of 10 years.')
+
+        c.argument('week_of_year',
+                   help='The Week of Year, 1 to 52, in which to take the yearly LTR backup.')
+
+    with self.argument_context('sql midb ltr-backup') as c:
+        c.argument('location_name',
+                   arg_type=get_location_type(self.cli_ctx),
+                   help='The location of the desired backup(s).',
+                   id_part=None)
+
+        c.argument('database_name',
+                   options_list=['--database', '-d'],
+                   id_part=None)
+
+        c.argument('managed_instance_name',
+                   options_list=['--managed-instance', '--mi'],
+                   id_part=None)
+
+        c.argument('backup_name',
+                   options_list=['--name', '-n'],
+                   help='The name of the LTR backup. '
+                   'Use \'az sql midb ltr-backup show\' or \'az sql midb ltr-backup list\' for backup name.',
+                   id_part=None)
+
+        c.argument('backup_id',
+                   options_list=['--backup-id', '--id'],
+                   help='The resource id of the backups. '
+                   'Use \'az sql midb ltr-backup show\' or \'az sql midb ltr-backup list\' for backup id. '
+                   'If provided, other arguments are not required. ')
+
+    with self.argument_context('sql midb ltr-backup list') as c:
```

Comment:
Please also add is_preview=True to new commands, which will give us buffer time to test command behavior.

Reply:
Sure;  added.  Is there a reason no other sql commands are in preview now?